test/functions.php

```diff
@@ -5,15 +5,94 @@
 
 class Functions extends TestCase
 {
-    public function testSuccessCreateBottlesHtml()
+    public function testSuccessCreateBottlesHtmlAddBottle()
```

Comment:
why AddBottle? this is just for when there is no bottle being edited, so can just be called `testSuccessCreateBottlesHtml`